label_studio/data_import/api.py

```diff
@@ -465,3 +470,43 @@ def get(self, *args, **kwargs):
             return RangedFileResponse(request, file.open(mode='rb'), content_type=content_type)
         else:
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+class DownloadStorageData(APIView):
+    """ Check auth for nginx auth_request
+    """
+    swagger_schema = None
+    http_method_names = ['get']
+    permission_classes = (IsAuthenticated, )
```

Comment:
should RBAC be presented here or how it works?

Reply:
@niklub  No, this call is available for all authenticated users, we have permissions check inside `get` code: 
```
if file_upload is not None and file_upload.has_permission(request.user):
```